lib/lrama/lexer.rb

```diff
@@ -62,8 +64,7 @@ def self.define_type(name)
     GrammarRules = 3
     Epilogue = 4
 
-    # Token types
-
+    # @dynamic prologue, bison_declarations, grammar_rules, epilogue, bison_declarations_tokens, grammar_rules_tokens
```

Comment:
In my environment `bundle exec steep check` passes without this annotation. Is there any benefit for these annotation?